org.osgi.test.cases.typedevent/bnd.bnd

```diff
@@ -9,8 +9,9 @@ Import-Package: ${-signaturetest}, *
 -buildpath: \
 	org.osgi.test.support;version=project,\
 	org.osgi.service.typedevent;version=latest,\
-	org.osgi.framework;maven-scope=provided;version=1.8,\
-	org.osgi.dto;maven-scope=provided;version=1.0
+	org.osgi.framework;maven-scope=provided;version='1.8',\
+	org.osgi.dto;maven-scope=provided;version='1.0',\
+	org.mockito.mockito-core
```

Comment:
You don't need to add mockito here since this project already includes osgi.tck.junit-platform.bnd

Reply:
thank you!